configure

```diff
@@ -126,6 +126,7 @@ done
 # Default settings
 #
 # Default lib behavior yes/no/auto
+_sdl=no
```

Comment:
Why not set here for 'auto' and remove this assignment in all those new places?

Reply:
If the default was set to auto, then an additional set of assignments would need to be added for the ports that don't use SDL. Personally, I think it would be better to have the default as it is at the moment.